config/secrets.js

```diff
@@ -0,0 +1,5 @@
+//create this file so we don't have to add it to routes file 
+
+module.exports = {
+    jwtSecret: process.env.JWT_SECRET || "I never watched Star Wars or Lord of The Rings"
+}
```

Comment:
I can't believe you never watched Star Wars or Lord of the Rings!!!! You are a developer!  These shows are mandatory nerd watching :)